consensus/src/replica.rs

```diff
@@ -281,6 +281,7 @@ impl<A: ConsensusApi<TYPES>, TYPES: NodeTypes> Replica<A, TYPES> {
         let mut last_view_number_visited = self.cur_view;
         let mut new_commit_reached: bool = false;
         let mut new_decide_reached = false;
+        let mut new_decide_qc = None;
```

Comment:
We could _technically_ initialize this to the genesis QC because that is the last decided view if there are no other decided views, but I don't think it is necessarily incorrect to do it this way either.  Just mentioning it.